text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+We hypothesize that implementing a folder tree structure using a template could help researchers collaborate on their projects,
+and manage their data and files better.
+In a first step, 
+we designed two templates (fig. 1) and a survey to guide our efforts.
+We are presenting here the analysis of the survey answer and a novel version of the template in accordance with the received feedback.
+This blog ends by outlining the gin-tonic application, that brings some technical solution (based on the git submodule technology) to add flexibility and ease of use in the template. 
+
+> *How do you organise your digital files for your research projects ?* 
+>
+> I organise my file using the GIN research repository (version 1).
+
+# The survey
+
+## Two template structure
+
+In order to obtain a practical flair in comments and feedback received,
+we provided two templates, with a similar number of folders,
+but organized differently.
+The 5_top structure was more hierarchical,
+while the 9_top structure was flatter (Fig. 1).
+In addition, we asked researchers to browse the folder tree while asking them to place or find specific files,
+hoping researchers will make themselves familiar with the template before giving us a detailed feedback. 
+
+![Fig 1: the two templates structure](../50_figures/pulication_ready_figures/2draft-templates.png)
+
+## No clear preference for one or the other template. 
+
+We got 51 responses to our survey.
+We prepared an analysis before the data collection and finally run it on the final data to produce a reproducible reports you can access here: XXX.
+In general, researchers reacted very positively to the project.
+Surprisingly, about half of the participants preferred one template, while the other half preferred the other one (Fig.2, 
+participants had to choose one or the other template).
+This preference  was highly correlated with the similarity of the template with the structure they were presently using (Pearson Chi-square, p−value = 2.23e−06),
+while we could see no visible effect of career stage or research domain on this preference,
+or any effect of this preference on willingness to use such a template.
+In particular, computer scientists seem not to differ from wet-lab researchers in these aspects.
+
+![Fig. 2: Wilingness to use a template, split by preference for one template.](../50_figures/pulication_ready_figures/fig2.png)
+
+We asked three questions about where they would save or search for specific documents in the different structures.
+Researchers were indicating different folders between them, and different than the ones we designed for it,
+showing that **a too detailed folder structure would not be helpful**.
+On the other hand, researchers would navigate the repository to find specific files using similar strategies (Fig. 3),
+suggesting that having a structure can be helpful, and reduce the time to browse for specific information.   
```

Comment:
suggesting that having a structure can be helpful, and may reduce the time to browse for specific information.